Formula/libbi.rb

```diff
@@ -0,0 +1,156 @@
+class Libbi < Formula
+  desc "Bayesian state-space modelling on parallel computer hardware"
+  homepage "http://libbi.org"
+  head "https://github.com/libbi/LibBi.git"
+
+  stable do
```

Comment:
`stable do` block is not needed. Just put `url` and `sha256` between `homepage` and `head`